docs/rules/require-expect.md

```diff
@@ -28,6 +40,28 @@ The "never-except-zero" option disallows `except` calls, except when used to
 explicitly assert that a test performs no assertions, which would otherwise
 be considered an error.
 
+### Config Options
+
+This rule allows exactly one config option: `allowFunctions`
+
+#### allowFunctions
+
+If you are using the Check Type `except-simple` whenever you pass `assert` into
+another function you will cause an error. If you have some functions that you
+are passing `assert` to that don't add to the number of asserts (and maybe just
```

Comment:
I think the part in parentheses is probably unnecessary. Would you mind removing it?